plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/internal/actions/layout/ArrangeBorderNodesAction.java

```diff
@@ -86,11 +85,6 @@ protected boolean isSelectionListener() {
         return true;
     }
 
-    @Override
-    protected void doRun(final IProgressMonitor progressMonitor) {
-        super.doRun(progressMonitor);
```

Comment:
not really necessary to do a dedicated commit

Reply:
Ok, merged with fixup